test/manyvariables.jl

```diff
@@ -6,6 +6,9 @@ using Test
 # using LinearAlgebra
 
 @testset "Test hash tables" begin
+    a = TaylorN{Float64}(9//10)
+    a isa TaylorN{Float64}
+    @test constant_term(a) == Float64(9//10)
```

Comment:
Could you add a test that checks that an error (`MethodError`) is thrown if you do not make explicit the "numtype" `T` of the `TaylorN{T}` when using this constructor?

Reply:
> Thanks a lot for this addition (and sorry for the time to send my review). In general, LGTM, though I have included a comment requesting to add another tests. Aside from that, once that is implemented, I'm happy to merge it.

Thanks for taking a look! I added your suggestion, so this is now ready for merging.